src/otserv.cpp

```diff
@@ -144,12 +150,13 @@ void mainLoader(int, char*[], ServiceManager* services)
 	std::cout << std::endl;
 
 	// check if config.lua or config.lua.dist exist
-	std::ifstream c_test("./config.lua");
+	const std::string& configFile = g_config.getString(ConfigManager::CONFIG_FILE);
+	std::ifstream c_test("./" + configFile);
 	if (!c_test.is_open()) {
-		std::ifstream config_lua_dist("./config.lua.dist");
+		std::ifstream config_lua_dist("./" + configFile + ".dist");
```

Comment:
This should not be variable. Since we know this file defaults to `config.lua.dist`, I think a more prudent thing is to leave it hardcoded.